webextensions/api/privacy.json

```diff
@@ -313,7 +313,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤77"
+                  "version_added": "20"
```

Comment:
✅ https://github.com/chromium/chromium/commit/75fc25972e9966a481e834dcd103de483bdffd99 landed in [20.0.1093.0](https://github.com/chromium/chromium/releases/tag/20.0.1093.0) before 20.0.1132.